spring-boot-admin/Dockerfile

```diff
@@ -36,5 +36,5 @@ RUN mkdir output && \
 FROM eclipse-temurin:21-alpine
 WORKDIR /springbootadmin
 COPY --from=build /scratch/output .
-EXPOSE 9090
+EXPOSE 9099
```

Comment:
Since there isn't a real standard port here, we _could_ let the server inside the container listen on 8080 and rely on documentation to suggest 9099 until/unless that one becomes a problem too, but it is nice to have the port being published match (`9099:9099` vs `9099:8080`)

Reply:
I'm fine either way.

Please open PRs to update https://github.com/SteeltoeOSS/Documentation/blob/v4/api/v4/management/springbootadmin.md and https://github.com/SteeltoeOSS/Samples/blob/latest/CommonTasks.md#spring-boot-admin for the changed ports.